dart/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   http: '>=0.13.0 <2.0.0'
   meta: ^1.3.0
   stack_trace: ^1.10.0
-  uuid: ^3.0.0
+  uuid: '>=3.0.0 <5.0.0'
```

Comment:
As far as I can tell, Sentry is not using any of the breaking changes, so we can keep supporting v3 while simultaneously supporting v4.